com.unity.render-pipelines.high-definition/Runtime/Utilities/ArrayUtils.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Unity.Collections;
+using UnityEngine.Jobs;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    internal static class ArrayUtils
+    {
+        public static void ResizeNativeArray<T>(ref NativeArray<T> array, int capacity) where T : struct
```

Comment:
is it a copy from URP? in this case I suggest we move this one in Core and remove it from URP

Reply:
Done, made this a public API on core.